javalib/src/main/scala/java/util/StringTokenizer.scala

```diff
@@ -1,103 +1,93 @@
 package java.util
 
-class StringTokenizer(string: String,
-                      private var delimiters: String,
-                      returnDelimiters: Boolean)
-    extends java.util.Enumeration[Object] {
+class StringTokenizer(
+  str: String,
+  private var delim: String,
+  returnDelims: Boolean
+) extends java.util.Enumeration[Object] {
 
-  def this(string: String) = this(string, " \t\n\r\f", false)
-  def this(string: String, delimeters: String) =
-    this(string, delimeters, false)
+  def this(str: String) = this(str, " \t\n\r\f", false)
+  def this(str: String, delim: String) = this(str, delim, false)
 
-  if (string == null) {
-    throw new NullPointerException()
-  }
-
-  private var position = 0
+  private var position: Int = 0
+  private val length: Int = str.length
 
   def countTokens(): Int = {
-    var count   = 0
-    var inToken = false
-    val length  = string.length
-    for (i <- position until length) {
-      if (delimiters.indexOf(string.charAt(i), 0) >= 0) {
-        if (returnDelimiters)
+    var count: Int = 0
+    var inToken: Boolean = false
+    var i: Int = position
+
+    while(i < length) {
+      if (isDelim(str.charAt(i))) {
+        if (returnDelims)
           count += 1
+
         if (inToken) {
           count += 1
           inToken = false
         }
       } else {
         inToken = true
       }
+
+      i += 1
     }
+
     if (inToken)
       count += 1
```

Comment:
I am pretty sure that the Scala Center style guide recommends that this be all on one line. I think a less preferred
option is enclosing it {} and placing on several lines.   The SN scalafmt config file seems to let this pass, so it is probably lore, which a more senior reviewer may override/reject.  I notice the same idiom a number of places in the file.

One line or not does not affect correctness, but an appropriate level of project uniformity makes it easier to
review thousands of lines of code.